src/Nox.Generator/Infrastructure/Persistence/DbContextGenerator/DbContextGenerator.cs

```diff
@@ -83,8 +83,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 }
             }
             
-            base.OnModelCreating(modelBuilder);
         }
+
```

Comment:
this changes did not produced a new SampleWeAppDbContext?

Reply:
Pushed generated code